tests/test_other.py

```diff
@@ -1162,7 +1163,7 @@ def test_dot_a_all_contents_invalid(self):
     # built by mistake
     create_file('native.c', 'int native() { return 5; }')
     create_file('main.c', 'extern int native(); int main() { return native(); }')
-    self.run_process([CLANG_CC, 'native.c', '-target', 'x86_64-linux', '-c', '-o', 'native.o'])
+    self.run_process([CLANG_CC, 'native.c', '-target', '%s-linux' % platform.machine(), '-c', '-o', 'native.o'])
```

Comment:
The `--target` flag can be completely removed once #16561 lands

Reply:
ack